reconcile/utils/gitlab_api.py

```diff
@@ -20,6 +21,9 @@
 
 MR_DESCRIPTION_COMMENT_ID = 0
 
+# The default value is there for unit test
+INTEGRATION_NAME = os.getenv("INTEGRATION_NAME", "qontract-reconcile")
```

Comment:
If you can't get the integration name, then I'd use an empty string based on the convention used elsewhere.

Reply:
Done.